Dockerfile

```diff
@@ -1,22 +1,33 @@
-FROM node:8
+FROM node:8-alpine
 
 WORKDIR /usr/src/app
 
-RUN apt-get update \
-    && apt-get install -y jq wget --no-install-recommends
+RUN apk --no-cache add \
+    bash \
+    g++ \
+    ca-certificates \
+    lz4-dev \
+    musl-dev \
+    cyrus-sasl-dev \
+    openssl-dev \
+    make \
+    python
+
+RUN apk add --no-cache --virtual .build-deps gcc zlib-dev libc-dev bsd-compat-headers py-setuptools bash git
```

Comment:
Aren't `g++`, `lz4-dev`, `musl-dev`, `cyrus-sasl-dev`, `openssl-dev`, `make` and `python` also build-only deps?

Reply:
They are but it was taking longer to figure out which ones of them are needed for runtime as `noderdkafka` was bailing out on startup. I figured that I can follow up later.